setup.py

```diff
@@ -3,12 +3,12 @@
 test_requires = [
     'pytest>=3.6.0',
     'pytest-django',
-    'django-webtest==1.9.3'
+    'django-webtest==1.9.3',
 ]
 
 setup(
     name='django-oscar-invoices',
-    version='0.1',
+    version='0.1.5',
```

Comment:
Why `0.1.5`, but not `0.2` or etc?

Reply:
well these are backwards compatible bugfixes and because of https://semver.org says they should increment the PATCH level. It took me 5 iterations to fix all the bugs, so there you go.